product_docs/docs/pem/8/index.mdx

```diff
@@ -54,6 +54,6 @@ redirects:
 - /pem/latest/pem_admin/01_overview/02_installing_pem_overview/
 ---
 
-Welcome to Postgres Enterprise Manager. PEM consists of components that provide the management and analytical functionality for your EDB Postgres Advanced Server or PostgreSQL database. PEM is based on the Open Source pgAdmin 4 project.
+Welcome to Postgres Enterprise Manager (PEM). PEM consists of components that provide the management and analytical functionality for your EDB Postgres Advanced Server or PostgreSQL database. (All PEM features are available for both EDB Postgres Advanced Server and PostgreSQL databases.) PEM is based on the Open Source pgAdmin 4 project.
```

Comment:
This point cannot be added here as we are not talking about the backend database here. We need to add this note where we are talking about the backend database.